src/tests/tracing/eventpipe/randomizedallocationsampling/manual/README.md

```diff
@@ -0,0 +1,112 @@
+# Manual Testing for Randomized Allocation Sampling
```

Comment:
What prevents these tests from being automated?

Reply:
The test as currently designed does large numbers of allocations with different types of objects and then prints statistical distributions to the console. Nothing fundamental prevents it from being converted to an automated test but the main factors that discourage it are:
- its a slow test even on a fast machine - it does large amounts of allocations to gather many statistical distributions
- it would require effort to convert it and ensure we've reasonably calibrated the sensitivity for the randomized results the test generates.

Historically our automated testing on tracing events validate that the events are generated and check some fields for reasonable data, but don't go into great detail. That level of testing has had good cost-benefit tradeoff in the past so we repeated it here.